document/lib/Pdf/Document/FontDict.hs

```diff
@@ -222,3 +231,85 @@ loadUnicodeCMap pdf fontDict =
             Left e -> throwIO $ Corrupted ("can't parse cmap: " ++ show e) []
             Right cmap -> return $ Just cmap
         _ -> throwIO $ Corrupted "ToUnicode: not a stream" []
+
+
+loadFontDescriptor :: Pdf -> Dict -> IO (Maybe FontDescriptor)
+loadFontDescriptor pdf fontDict = do
+  case HashMap.lookup "FontDescriptor" fontDict of
+    Nothing -> return Nothing
+    Just o -> do
+      ref <- sure $ refValue o
+             `notice` "FontDescriptor should be a reference"
+      fd <- (sure . (`notice` "FontDescriptor: not a dictionary") . dictValue) =<<
+            lookupObject pdf ref
+
+      fontName <- required "FontName" nameValue' fd
+      fontFamily <- optional "FontFamily" (fmap decodeUtf8 . stringValue) fd
```

Comment:
`FontFamily` is a byte string, it doesn't have any encoding, see section 7.9.2.4 of the specification. Lets keep it as a `ByteString`.

Reply:
Yes, again I've changed it.